test/common/index.js

```diff
@@ -802,3 +804,18 @@ exports.hijackStdout = hijackStdWritable.bind(null, 'stdout');
 exports.hijackStderr = hijackStdWritable.bind(null, 'stderr');
 exports.restoreStdout = restoreWritable.bind(null, 'stdout');
 exports.restoreStderr = restoreWritable.bind(null, 'stderr');
+
+exports.getTTYfd = function getTTYfd() {
+  const tty = require('tty');
+  // Do our best to grab a tty fd.
+  const ttyFd = [1, 2, 4, 5].find(tty.isatty);
```

Comment:
`0` is also a very good candidate for this

Reply:
That was removed recently: https://github.com/nodejs/node/commit/ef2861961c3d9e9ed6972e1e84d969683b25cf95

Reply:
Interesting. I'll have to leave a note about this.